roadmap.md

```diff
@@ -4,154 +4,95 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-The roadmap below outlines KubeEdge’s 2023 feature plan.
+The roadmap below outlines KubeEdge’s 2024 feature plan.
 
-## 2023 H1 
+## SIG Node
 
-### SIG Node
+- Continuous follow up Kubernetes release
+- Support edge nodes running on mac OS and RTOS 
+- Support in-cluster config for edge pods to access Kube-APIServer
+- Enhancements to device plugin on edge nodes, such as support for multiple virtual GPUs
+- Support event report to cloud 
+- Support for serverless computing
+- Remote maintenance
+- Add edge nodes in batches
 
-- Support WasmEdge integration on KubeEdge edgenode
-- Support Kubectl attach to container running on edgenode
-- Update Kubernetes dependency to v1.24.14, switch default container runtime at edge to containerd
+## SIG Device-IOT
 
-### SIG Device-IOT
+- Multi-language Mappers support
+- Devices discovery 
```

Comment:
I would like to know the overview for this, sounds really useful to manage leaf devices?

Reply:
> I would like to know the overview for this, sounds really useful to manage leaf devices?

I understand it's to enable automatic discovery and connection with edge nodes so that devices can be seamlessly migrated to other nodes. https://github.com/kubeedge/kubeedge/issues/4744